### ISSUE TITLE
specifying bounds for subplots

### ISSUE THREAD

**Issue by heistp** [tohojo/flent#206]
In a plot that has subplots, like all_scaled, it appears that --bounds-y can only be specified once, and it applies to all of the subplots for the plot. Is there a way to specify bounds separately for each subplot?

**Comment by tohojo** [comment]
Nope, there currently isn't. But now that you mention it, there probably should be :)

How about just extending the existing syntax by supporting more args. I.e., for a plot with subplots, the first subplot would get the first two --bounds-{x,y} args, the second subplot would get the next two, etc. And if only two were specified, they would still be propagated across all subplots to keep compatibility with the current behaviour?

**Comment by heistp** [comment]
That makes sense to me, and should be supported in the gui with no changes? So far this isn't urgent for me now so I probably won't get to it soon, so either you can do it if you have time and it's easy, or else I'll come back to it when needed later. :)
